scss/tech.scss

```diff
@@ -26,17 +26,23 @@ $text-small: 12px;
   text-align: center;
   width: 200px;
 
-   &::before {
-     border: 1px solid $green;
-     border-radius: 8px;
-     right: 10px;
-     bottom: 10px;
-     content: '';
-     height: 100%;
-     position: absolute;
-     width: 100%;
-     z-index: -1;
-   }
+  &::before {
+    border: 1px solid $green;
+    border-radius: 8px;
+    right: 10px;
+    bottom: 10px;
+    content: '';
+    height: 100%;
+    position: absolute;
+    transition: bottom 100ms ease-out;
+    width: 100%;
+    z-index: -1;
+  }
+}
+
+.box:hover::before {
```

Comment:
this uses `.box` which is declared in line 16 you can put it inside of that one and do: `&:hover::before`